### PR TITLE
fixes #3115

### LINKED ISSUE #3115: Migration Guide: Correct typo for deep-assign
There is a typo in the docs: https://github.com/canjs/canjs/blame/master/docs/can-guides/commitment/migrating_to_3.0.md#L300

It should instead be import deepAssign from "can-util/js/`deep-assign/deep-assign`";

### PR DESCRIPTION
**Pull request by GeneralMeow**

### CODE REVIEW

**Comment by matthewp** [comment]
👍

**Comment by GeneralMeow** [comment]
should i get rid of my fix for #3117 ? doesn't seem to be passing the tests

File: docs/can-guides/components/todomvc-light.html [hunk 1]
```diff
@@ -179,7 +179,7 @@
 			el.children('.edit').focus();
 		},
 		updateTodo: function (todo, el) {
-			var value = can.trim(el.val());
+			var value = String.prototype.trim(el.val());
```
Comment (chasenlehara) [inline docs/can-guides/components/todomvc-light.html]: Should this be `String.prototype.trim.call(el.val())`?

Comment (chasenlehara) [inline docs/can-guides/components/todomvc-light.html]: Also, I wonder if any protections for `null` or `undefined` need to be added… `String.prototype.trim.call` will throw if it’s given either of those values.

**Comment by GeneralMeow** [comment]
Thank you so much for the feedback. I will incorporate it and get the PR updated!

File: docs/can-guides/components/todomvc-light.html (hunk 1 above)
Comment (GeneralMeow) [inline docs/can-guides/components/todomvc-light.html]: should i add .call on all the other String.prototype.trim functions as well?

**Pull request closed without merging**